UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.Common/HealthCheck/IAzureWebJobHelper.cs

```diff
@@ -0,0 +1,11 @@
+﻿
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace UKHO.ExternalNotificationService.Common.HealthCheck
+{
+    public interface IAzureWebJobHelper
+    {
+        public Task<HealthCheckResult> CheckWebJobsHealth(WebJobDetails webJob);
+    }
```

Comment:
Sort Usings.
Access modifiers not required in interface for methods

Reply:
Remove Access modifires